src/modules/endpoint/fetchers/categories.js

```diff
@@ -0,0 +1,71 @@
+// @flow
```

Comment:
I really liked the Endpoint.js and Endpoint.js builder to give it more structure. There's a lot of freedom you have now and not a real schema. For example you have to call `.json()` a lot and have to come up with a way of mapping urls with `urlMapper`

Reply:
imo Endpoint and Endpoint builder made everything way more complex than it had to be, I didn't like it. We could discuss this though and maybe readd them with another task. This one is already big enough.